src/test/java/login/LoginInteractorTest.java

```diff
@@ -0,0 +1,142 @@
+package login;
+
+import entity.AbstractUser;
+import exception.AuthException;
+import mock.MockLoginPresenter;
+import mock.MockUserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import service.login.LoginInteractor;
+import service.login.LoginInputData;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Test suite for the LoginInteractor.
+ */
+public class LoginInteractorTest {
+    private MockUserRepository mockUserRepository = new MockUserRepository();
+    private MockLoginPresenter mockLoginPresenter = new MockLoginPresenter();
+    private LoginInteractor loginInteractor = new LoginInteractor(mockUserRepository, mockLoginPresenter);
+
+
+    @BeforeEach
+    public void setUp() {
+        mockUserRepository = new MockUserRepository();
+        mockLoginPresenter = new MockLoginPresenter();
+        loginInteractor = new LoginInteractor(mockUserRepository, mockLoginPresenter);
+    }
+
+    /**
+     * Test login with an invalid email format.
+     */
+    @Test
+    public void testLogin_InvalidEmail() {
```

Comment:
general naming format for tests in this case is

testLoginInteractorInvalidEmail. Don't use underscores, camel case instead.